### DISCUSSION TITLE
the icon style of expander looks different in version 11.0-rc

### DISCUSSION THREAD

**Discussion by jeff9571** [AvaloniaUI/Avalonia#11980] [Q&A] [answered]
The expander looks different from running status to the previewer.
Here is the previewer mode:
![02150677127afdf3e038866b775b076](https://github.com/AvaloniaUI/Avalonia/assets/35749706/d09c87f3-93e8-4d96-aa4f-b9310e33f5b2)
And here is after running the app:
![b89113ecd6f746dfe68ec83eb002372](https://github.com/AvaloniaUI/Avalonia/assets/35749706/2edba820-4331-4403-8967-caf8d1192996)
The codes:
` <Expander Header="Content" Width="{Binding #ThirdPanel.Bounds.Width}" IsExpanded="True"
                                          Height="200"></Expander>`
using avalonia version 11.0-rc1.1

**Comment by rabbitism** [comment]
Are you using any additional theme?

**Reply by jeff9571** [reply]
no
![b36ddf7cb81f7c5861016b685fe4666](https://github.com/AvaloniaUI/Avalonia/assets/35749706/2f41594f-f8cd-4a32-8d93-3c378ca22986)

**Reply by timunie** [reply, marked as answer]
can't reproduce using RC2.2. Please provide a minimal sample to check. 

Note: Somehow, it looks like you have a combination of simple theme and fluent theme? If so this may be the root cause.
